packages/keyring-eth-ledger-bridge/src/ledger-keyring.ts

```diff
@@ -551,6 +551,7 @@ export class LedgerKeyring extends EventEmitter {
   }
 
   forgetDevice(): void {
+    this.deviceId = '';
```

Comment:
should this be `null` instead?

Reply:
I've seen other mention to `''` when deserializing

Reply:
Well, its initial state is an empty string, so I don't know if we want to change on both sides?